cirq/ops/parity_gates.py

```diff
@@ -21,7 +21,8 @@
 from cirq import protocols
 from cirq._compat import proper_repr
 from cirq.ops import gate_features, eigen_gate
-from cirq.ops.common_gates import _rads_func_symbol
+from cirq.ops.common_gates import _rads_func_symbol, ZPowGate, CZPowGate
+from cirq.ops.global_phase_op import GlobalPhaseOperation
```

Comment:
You didn't change these imports to `... import common_gates, global_phase_op`.

Reply:
Changed.  Is this what you mean now?

Reply:
Yes.  Thanks.